ConfigurationSystem/Agent/VOMS2CSAgent.py

```diff
@@ -18,11 +16,29 @@
                                                               getAllUsers
 from DIRAC.Core.Utilities.Proxy                        import executeWithUserProxy
 
+__RCSID__ = "$Id$"
+
 class VOMS2CSAgent( AgentModule ):
 
-  def initialize( self ):
+  def __init__( self, *args, **kwargs ):
+    """ Defines default parameters
+    """
 
     self.__voDict = {}
+    self.__adminMsgs = {}
+    self.csapi = None
+    self.voChanged = False
+    self.dryRun = True
```

Comment:
I would set this to `False`, or people updating will not notice that their agents are now behaving differently, and existing agents don't have the option set in the CS.